pages/treeherder.py

```diff
@@ -51,12 +70,35 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def open_repos_menu(self):
+        self.selenium.find_element(*self._repos_menu_locator).click()
+
+    def open_single_resultset(self):
+        Wait(self.selenium, self.timeout).until(
+            EC.visibility_of_element_located(self._first_resultset_datestamp_locator))
+        self.selenium.find_element(*self._first_resultset_datestamp_locator).click()
+
     def pin_using_spacebar(self):
         el = self.selenium.find_element(*self._result_sets_locator)
         Wait(self.selenium, self.timeout).until(EC.visibility_of(el))
         el.send_keys(Keys.SPACE)
         Wait(self.selenium, self.timeout).until(lambda _: self.pinboard.is_pinboard_open)
 
+    def select_mozilla_central_repo(self):
```

Comment:
I'd prefer a method that took the repository name to switch to, but I'm okay with this as it's hopefully temporary anyway.